From f038049b2025ca7fe349bf715a40b4f956fba9ac Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Tue, 18 Apr 2017 09:44:21 +0800
Subject: [PATCH 1703/5242] MLK-14704: ARM: imx: pm-rpmsg: correct A7 mode
 after resume back

commit  c7d55a7689d1e920c6f7e442d3e0f3fbb24d4c9a from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct right RUN mode after A7 resume back instead of the VLLS mode

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
(cherry picked from commit f6b3dda9babd17fcbab5fc2b05c3f64660ea6250)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/pm-rpmsg.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/pm-rpmsg.c b/arch/arm/mach-imx/pm-rpmsg.c
index 9407929..b4bfa2a 100644
--- a/arch/arm/mach-imx/pm-rpmsg.c
+++ b/arch/arm/mach-imx/pm-rpmsg.c
@@ -291,7 +291,7 @@ static int pm_heartbeat_resume(struct device *dev)
 {
 	int err;
 
-	err = pm_vlls_notify_m4(true);
+	err = pm_vlls_notify_m4(false);
 	if (err)
 		return err;
 
-- 
1.7.9.5

